addOns/alertFilters/src/main/java/org/zaproxy/zap/extension/alertFilters/DialogAddAlertFilter.java

```diff
@@ -273,11 +273,11 @@ public void setAlertFilter(AlertFilter alertFilter) {
     }
 
     /**
-     * Gets the alertFilter defined in the dialog, will be null if the dialog is cancelled.
+     * Gets the {@code AlertFilter} defined in the dialog, will be null if the dialog is cancelled.
      *
-     * @return the alertFilter, if correctly built or null, otherwise
+     * @return the {@code AlertFilter}, if correctly built or null otherwise
      */
-    public AlertFilter getAlertFilter() {
+    public AlertFilter getUpdatedAlertFilter() {
```

Comment:
Better rename the field than this method IMO.

Reply:
The class has two variables, hence the distinction.
```
    private AlertFilter alertFilter;
    private AlertFilter updatedAlertFilter;
```

Reply:
Yes, but there's no getter for the other (e.g. `alertFilter` would be better as `previousAlertFilter` or `oldAlertFilter`).

Reply:
Oh okay, I see what you mean now.